src/endpoint/s3/ops/s3_put_bucket_policy.js

```diff
@@ -10,19 +10,18 @@ const S3Error = require('../s3_errors').S3Error;
 async function put_bucket_policy(req) {
     let policy;
     try {
-        policy = JSON.parse(JSON.stringify(req.body).toLowerCase());
```

Comment:
IIUC, by removing this you make the bucket policy case matching sensitive. is this intended? what should be the correct behavior?

Reply:
yes, this is intended AWS bucket policies are case-sensitive. I have confirmed it by trying to push policies with correct and incorrect casing to AWS bucket policies